docs/proposal/adsFlush.md

```diff
@@ -44,11 +44,13 @@ Kmesh is synchronised with the istiod control surfaces through the six steps des
 
   When the bpf map is being updated, the xDS controller will continue processing configuration update messages from Istiod and update `xDS.Cache`. After each bpf map update is completed, it will check `xDS.Cache` through a semaphore to see if it has been updated. If updated, it will make a deep copy of `xDS.Cache` and continue updating the bpf map.
 
-![alt text](pics/map_flush.svg)
+<div align="center">
+<img src="pics/map_flush.svg" width="800" />
+</div>
 
 As shown above, the Ads Controller continues to update the `xDS.Cache` at the time of the bpf map update.
 
-When the map flush controller starts a new processing, it first checks `xDS.cache` through a semaphore to see if it has changed. If it has changed, it will make a deep copy of `xDS.Cache`.
+When the map flush controller starts a new processing, it first checks `xDS.cache` through a channel to see if it has changed. If it has changed, it will make a deep copy of `xDS.Cache`.
```

Comment:
so this is a one notify one update ops?  But as we have discussed, if multi notifications in the channel, actually we only need to do one time bpf map update. Because the xds.Cache has already been updated with the newest status